tests/ui/sanitizer/cfi/coroutine.rs

```diff
@@ -26,7 +26,7 @@ use std::async_iter::AsyncIterator;
 
 #[test]
 fn general_coroutine() {
-    let mut coro = #[coroutine] |x: i32| {
+    let coro = #[coroutine] |x: i32| {
```

Comment:
`mut` was never required here. The only change is that before this PR, the compiler did not warn that `mut` wasn't necessary.